backend/src/routes/preferences.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+var mongoose = require('mongoose');
+const Preferences = require("../models/preferencesModel");
+const router = express.Router();
+
+/**
+ * Get Preferences Object from Preferences Collection by Id of respective user
+ * 
+ * Route: GET /api/user/:userId/preferences where userId is the ID of user
+ */
+router.get('/:userId/preferences', async (req, res) => {
+    try {
+        const preferences = await Preferences.findOne({userID: req.params.userId});
+        if(preferences){
+            res.status(200).json(preferences);
+        }else{
+            res.status(404).json({"status": 404, "message": "Did not match any user!"});
```

Comment:
I think having a `status` field is redundant though, since the response code already includes it.